examples/original224.yaml

```diff
@@ -1,6 +1,7 @@
-- ladino: Al lugar de azer una koza serioza se esta okupando de bavajadas
-  words:
-    - bavajada
```

Comment:
please do not remove the words: entries, they indicate the origins of the egzamples.